src/main/resources/scripts/equip anything.sk

```diff
@@ -5,18 +5,9 @@
 aliases:
 	custom helmets = iron block, gold block, diamond block # Change this to anything you'd like
 
-on inventory click:
```

Comment:
The whole point of the example script was this part, why did you remove it?

Reply:
it didn't work so I removed it as I couldn't figure out how to fix it.

Reply:
How does it not work... ?

Reply:
idk, it just didn't, atleast on the latest release, I didn't wanna test on a older release (dev33) incase of fixed bugs or something.

Reply:
That would be a rather high priority bug, then. Ouch.

Reply:
I tried it on dev33 as well and it didn't work, so what do you want me to do? Open an issue or what?

Reply:
Will do.

Reply:
Created #1210 .